meeting-notes/2018/2018-08-20--go-core-dev-team-weekly.md

```diff
@@ -0,0 +1,204 @@
+# 💫Golang Core Dev Team Weekly Sync 🙌🏽 August 20, 2018
+
+- **Lead:** Dominic Della Valle
+- **Notetaker:** 
+  - @bigs
+- **Attendees:**
+  - @mgoelzer
+  - @keks
+  - @djdv
+  - @stebalian
+  - @kevina
+  - @hsanjuan
+  - @Magik6k
+- **Recording:** _add link to recording once it's online_
+
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- Round of updates
+  - What have you accomplished since the last Weekly?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+- Plan this week
+  - Select issues to work on
+- Review remaining issues if there is time left
+
+
+## Notes
+
+### Week Update
+
+@name
+ - Done:
+ - Blocked:
+ - Next:
+ 
+@djdv
+ - Done:
+   - Refined the IPFS on fuse implementation
+     - Added caching around things like pins, keys, and metadata
+     - Reduced lock contention
+     - Reworked how indexes are generated and referenced
```

Comment:
@djdv Can you do a demo of it working on Windows? @hacdias will be excited about it for IPFS Desktop for sure and @alanshaw will be able to give it a new spin with https://www.npmjs.com/package/ipfs-fuse

Reply:
I have an informal demo of the original implementation here: https://github.com/ipfs/go-ipfs/issues/5003#issuecomment-412151849
The refinement is the same, just much faster.
I plan to do a proper demo when this is closer to completion (still working on write support).